### ISSUE TITLE
Training Data Preprocessing

### ISSUE THREAD

**Issue by NuoWenLei** [NuoWenLei/deep-learning-final-project#2]
### Dataset Encoding

- Encode all images in episode with PRE-TRAINED ENCODER
- Save encoded latents in .npy file

### Dataset Loader

- (If size of entire dataset is very large) create batch train loader that loads batches at a time
- (If not) create data generator function that infinitely loads new data and shuffles on each epoch
